apps/mobile/src/screens/NftDetailScreen/NftAdditionalDetailsEth.tsx

```diff
@@ -108,7 +108,7 @@ export function NftAdditionalDetailsEth({ tokenRef }: NftAdditionalDetailsEthPro
           {projectUrl && (
             <DetailSection>
               <DetailLabelText>VIEW ON</DetailLabelText>
-              <View className="flex flex-col pt-4">
+              <View className="flex flex-col">
                 <DetailExternalLink
```

Comment:
this is a fix for previous mobile nft details regression:
<img width="471" alt="Screenshot 2023-08-02 at 6 05 28 PM" src="https://github.com/gallery-so/gallery/assets/49758803/d36b195c-6b5c-4098-8567-5c1685f9785a">

Reply:
gorgeous